setup.py

```diff
@@ -51,46 +58,78 @@ def link_directories(repo_path):
     print('Linking "{}" to "{}"'.format(repo_path, symlink_path)) 
     os.symlink(repo_path, symlink_path)
 
+
 def clone_repo(clone_path, repository):
     repo_name = "potato_" + repository
     repo_path = os.path.join(clone_path, repo_name)
 
-    subprocess.run(['git', 'clone', BW_GITHUB_PATH.format(repository), repo_path])
-    link_directories(repo_path)
+    try:
+        subprocess.run(['git', 'clone', BW_GITHUB_PATH.format(repository), repo_path],
+                       check=False)
+        link_directories(repo_path)
+    except:
+        print("Error cloning {} to {}!".format(
+            BW_GITHUB_PATH.format(repository), repo_path))
+
 
 def create_repo(clone_path, repository):
     print("Not implemented")
 
+
 def link(args):
     link_directories(args.repositories[0])
-    
+
+
 def add(args):
     args.path = os.path.join(args.path, 'potato_field')
 
     for repo in args.repositories:
-        clone_repo(args.path, repo)
-
-if not has_requirements():
-    exit()
-
-parser = argparse.ArgumentParser(
-        prog = 'setup.py',
-        description = 'Manages the setup of any repository that falls under potato_field',
-        epilog = """
-    potato_field_manager  Copyright (C) 2022  Bailey Danyluk
-    This program comes with ABSOLUTELY NO WARRANTY.
-    This is free software, and you are welcome to redistribute it
-    under certain conditions."""
-)
-
-parser.add_argument('mode', choices=['link', 'add'], default='add', help='Whether to setup a new repository or link an existing one')
-parser.add_argument('repositories', nargs='+', help='Select which repositories to clone and setup')
-parser.add_argument('--path', '-p', required=False, type=is_path, default=os.path.dirname(os.path.realpath(__file__)), help='The directory where the repository will be cloned to. If not specified, repository will be cloned within the directory this tool is run')
-args = parser.parse_args()
-
-if args.mode == 'link':
-    link(args)
-elif args.mode == 'add':
-    add(args)
-
-
+        if args.new:
+            create_repo(args.path, repo)
+        else:
+            clone_repo(args.path, repo)
+
+
+def main():
+    if not has_requirements():
+        sys.exit(1)
```

Comment:
I know you didn't make this, but can you add a print that says that the requirements are not fulfilled? I want to later add specifics in _which_ requirements, but I do want users to know why it failed